Khan/perseus
# [Locked Figure Labels] Add/edit/delete locked polygon labels

## Summary:
Add the ability to add/edit/delete labels for locked polygons
within the interactive graph editor.

Issue: https://khanacademy.atlassian.net/browse/LEMS-2350

## Test plan:
- Go to http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--mafs-with-locked-polygon-labels-flag
- Go to locked figures
- Open the locked polygon settings
- Confirm that the label settings are there
- Play around with the label settings and confirm that
  the label moves and updates with the polygon

<img width="399" alt="image" src="https://github.com/user-attachments/assets/471d586e-be2f-488f-9398-5b352e1403cc">
